tests/automation/linked_device_requests.spec.ts

```diff
@@ -11,68 +9,81 @@ import {
   waitForTextMessage,
 } from './utilities/utils';
 
-sessionTestTwoWindows('Accept request syncs', async ([windowA, windowB]) => {
-  const [userA, userB] = await Promise.all([
-    newUser(windowA, 'Alice'),
-    newUser(windowB, 'Bob'),
-  ]);
-  const [windowC] = await linkedDevice(userB.recoveryPhrase);
+sessionTestThreeWindowsWithTwoLinked(
+  'Accept request syncs',
+  ['Alice', 'Bob'],
+  async (
+    { windowsLinked: [windowA, windowB], otherWindow },
```

Comment:
Not sure about otherWindow, don't feel like that's very clear

Reply:
would you prefer those to be named by usernames?
    { windowsLinked: [Alice1, Alice2], Bob1 },

Reply:
issue with windowA, windowB and windowC is that sometimes they are not what we think. 

I mean, if you are given windowA, windowB and windowC, everytime you want to do something on it you are like, ok, I need the second window from the linked device, which was windowB.
If we name them better, there is no need for this anymore and it's far cleaner.

Reply:
I think Alice1, Alice2, Bob is the way to go, and we remove the usernames from the fixture so they make sense.

Reply:
made the change for that

Reply:
I thought I replired to this already but
`alice1` is alice's first window
`alice2` is alice's second window (linked device
`alice` is the user itself (with recovery/usernam/sessionId fields)